packages/jest-test-sequencer/src/__tests__/test_sequencer.test.ts

```diff
@@ -56,7 +57,10 @@ const toTests = (paths: Array<string>) =>
 
 beforeEach(() => {
   jest.clearAllMocks();
-  sequencer = new TestSequencer();
+  sequencer = new TestSequencer({
+    contexts: [],
+    globalConfig: {} as Config.GlobalConfig,
+  });
```

Comment:
Not sure if this is ideal, but since the Jest `TestSequencer` doesn't use either contexts, or globalConfig, it should be fine to have stand ins here, for now.

Alternatively we could make `TestSequencerOptions` optional, or even properly create contexts and globalConfig in the beforeEach.

lmk what you think.